HELP.md

```diff
@@ -0,0 +1,28 @@
+# Getting Started
```

Comment:
Not sure why .gitignore was not respected. We don't need to include this file in our commit.